packages/react-core/src/helpers/OUIA/OUIA.md

```diff
@@ -0,0 +1,93 @@
+---
+id: Open UI Automation
+section: developer-resources
+---
+
+## About Open UI Automation
+
+Open UI Automation (OUIA) is a specification of certain key guidelines to follow when creating a new
+web framework or application. The goal is to ease the burden of individuals wishing to create and
+maintain automated testing environments. OUIA standardizes some HTML attributes and web page
+behavior. It helps to create trivial and unambiguous locators of the elements in the DOM for
+various automated test suites.
+
+### Why not to use `id`, `ARIA` etc.?
+
+`ARIA` is designed as an accessibility specification and it doesn't account aspects that are crucial
+for automation such as a component type. `id` attribute might be reserved by some frontend code and
```

Comment:
```suggestion
for automation such as a component type. `id` attribute might be reserved by some front-end code and
```

Reply:
Added hyphen to align with how RH documents "front-end" as an adj. in the corporate style guide.